app/controllers/art_items_controller.rb

```diff
@@ -96,13 +93,22 @@ def set_art_item
     end
 
     def set_departments_list
-      @departments_list = Department.all
+      @departments_list = Department.where(department_id: current_user_departments)
     end
 
     def set_appraisal_types
       @appraisal_types = AppraisalType.all
     end
 
+    def get_artitems_collection
+      ArtItem.active_with_departments.where(department_id: current_user_departments)
+    end
+     
+    def check_for_authorized_access
```

Comment:
Consider moving this method inside ApplicationController along with other helper methods.

Reply:
I did move it